src/components/genesys-web-message/GenesysWebMessage.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="d-none"></div>
+</template>
+
+<script lang="ts">
+// Libraries
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+/**
+ * This component creates the button to generate a chat with Service BC Helpdesk.
+ * This component cannot be unit tested because it heavily relies on a third-party library.
+ * @property {string} deploymentKey - the deployment key (ID)
+ */
+@Component({})
+export default class GenesysWebMessage extends Vue {
+  @Prop({ default: '0' }) readonly deploymentKey!: string
+
+  mounted (): void {
+    if (this.deploymentKey !== '0') {
+      this.initWebMsg(
+        window,
+        'Genesys',
+        'https://apps.cac1.pure.cloud/genesys-bootstrap/genesys.min.js',
+        {
+          environment: 'cac1',
```

Comment:
I think the URL and environment would use props, too -- for more future flexibility.

Reply:
I actually had them there. I removed them after our meeting with them. I'll re-add them then!